sql-statements/sql-statement-alter-resource-group.md

```diff
@@ -53,6 +53,8 @@ ResourceGroupRunawayOptionList ::=
 
 DirectResourceGroupRunawayOption ::=
     "EXEC_ELAPSED" EqOpt stringLit
+|   "PROCESSED_KEYS" EqOpt intLit
+|   "REQUEST_UNIT" EqOpt intLit
```

Comment:
After discussion with PM, we will change `REQUEST_UNIT` to `RU` in sql syntax.
Please refer to this commit to make change, thx~
https://github.com/pingcap/docs-cn/pull/18575/commits/0e20a7de9adcb1e6f75a9400dcff82eb786deacc

Reply:
Got it.

Reply:
Done in https://github.com/pingcap/docs/pull/18858/commits/7178d520e585cae1414d12e081d196c4d098721f